### PR TITLE
Button & image

### PR DESCRIPTION
**Pull request by dev-ashishsethi**
Please provide your feedback

### CODE REVIEW

File: component/image/image.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+        <div class="main-content">
+
+            <h1>Images</h1>
+            <p>Buttons are usually used for performing any action.</p>
+            <h2>Responsive Image</h2>
+            <p> You can use responsive image in your project using below code. Image will have 100% of width of container and height will adjust accordingly while reserving aspect ratio. <br>
+            Note: <strong>In below example you can use scroll bar to resize the image container therefore resizing the image.</strong></p>
+            <div class="img-container">
+            <img src="..\..\assets\images\WALLPAPER.JPG" alt="" class="img-responsive">
```
Comment (dhimanaishwarya) [inline component/image/image.html]: you can add alt for images

**Review by m26unkwn — APPROVED** [review]
Code is looking good to me.

Good to merge

**Review by PuneetP16 — APPROVED** [review]
Looks good to merge.

File: component/image/image.html (hunk 1 above)
Comment (dev-ashishsethi) [reply component/image/image.html]: Thanks for suggestion!

**Pull request merged**